benchexec/tablegenerator/react-table/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The point of having an opinionated code formatter is to have no need for thinking about and discussing its configuration. In fact, a good opinionated code formatter would not even provide options at all (e.g., google-java-format). So I would prefer to keep the default config unless there are really strong arguments.

A line length of 160 is way too much anyway, line lengths should be kept to 80-100 characters. But the default line length is better than others.